src/Simplex/Messaging/Server/Main.hs

```diff
@@ -44,62 +56,99 @@ smpServerCLI cfgPath logPath =
     defaultServerPort = "5223"
     executableName = "smp-server"
     storeLogFilePath = combine logPath "smp-server-store.log"
-    initializeServer InitOptions {enableStoreLog, signAlgorithm, ip, fqdn} = do
-      deleteDirIfExists cfgPath
-      deleteDirIfExists logPath
-      createDirectoryIfMissing True cfgPath
-      createDirectoryIfMissing True logPath
-      let x509cfg = defaultX509Config {commonName = fromMaybe ip fqdn, signAlgorithm}
-      fp <- createServerX509 cfgPath x509cfg
-      writeFile iniFile iniFileContent
-      putStrLn $ "Server initialized, you can modify configuration in " <> iniFile <> ".\nRun `" <> executableName <> " start` to start server."
-      printServiceInfo serverVersion fp
-      warnCAPrivateKeyFile cfgPath x509cfg
+    initializeServer opts
+      | scripted opts = initialize opts
+      | otherwise = do
+        void $ withPrompt "SMP server will be initialized (press Enter)" getLine
+        enableStoreLog <- onOffPrompt "Enable store log to restore queues and messages on server restart" True
+        logStats <- onOffPrompt "Enable server storing server statistics daily" False
+        putStrLn "Should server require a password to create new messaging queues?"
+        password <- withPrompt "Enter 'r' for random (default), 'n' - no password, or enter password: " serverPassword
+        let host = fromMaybe (ip opts) (fqdn opts)
+        host' <- withPrompt ("Enter server FQDN or IP address for certificate (" <> host <> "): ") getLine
```

Comment:
```suggestion
        host' <- withPrompt ("Enter server FQDN or IP address for certificate (default " <> host <> "): ") getLine
```

Reply:
it's only going to be if passed via options, or it'll be 127.0.0.1